### PR TITLE
refactor scalings

### PR DESCRIPTION
**Pull request by ferrine**
according to experience in #2097

### CODE REVIEW

**Comment by aseyboldt** [comment]
Nitpick: No empty lines between parameters and between titles and parameter lists: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt

**Comment by ferrine** [comment]
I'll make docs neat then

File: pymc3/variational/approximations.py [hunk 1]
```diff
@@ -377,14 +405,18 @@ def sample_approx(approx, draws=100, hide_transformed=False):
 
     Parameters
     ----------
+    
     approx : Approximation
+
```
Comment (twiecki) [inline pymc3/variational/approximations.py]: I don't think we need new-lines in-between arugments.

**Comment by ferrine** [comment]
I've found one more theano bug that is hard to reproduce
setting `cost_part_grad_scale` to shared and `n_obj_mc>1` leads to strange error. I'll open an issue for it

**Comment by twiecki** [comment]
@ferrine So this is still WIP?

**Comment by ferrine** [comment]
No that's fine to merge. I'll do it today.

**Pull request merged**